packages/samples/headless-react/src/components/context/coveo-context.ts

```diff
@@ -0,0 +1,10 @@
+import {buildContext} from '@coveo/headless';
+import {engine} from '../../engine';
+
+export function setCoveoContext(pageId: string) {
```

Comment:
I think we should remove `Coveo` from the name? It feels inconsistent to mention it here, and not for other examples? (e.g. `buildCoveoSearchBox`) ?